coremltools/models/utils.py

```diff
@@ -2252,3 +2252,138 @@ def _make_second_chunk_prog(prog: _mil.Program, op_idx: int) -> _mil.Program:
     return prog
 
 
+def change_output_tensor_type(
+    ml_model: "_ct.models.model.MLModel",
+    from_type: int,
+    to_type: int,
+    function_names: _Optional[_List[str]] = None,
+) -> "_ct.models.model.MLModel":
+    """
+    Change output type of the CoreML model tensor outputs. Supported types are FLOAT16, FLOAT32.
```

Comment:
nit: "CoreML" -> "Core ML"

Reply:
It would be good to make this change in your user guide updates too.

Reply:
updated!